### DISCUSSION TITLE
Squash automerge - include PR number

### DISCUSSION THREAD

**Discussion by elijaholmos** [renovatebot/renovate#18215] [Request Help] [answered]
I've noticed that in this repository, renovate PRs that are squashed and merged include the PR number at the end of the commit message, which is indeed standard: 
![image](https://user-images.githubusercontent.com/35435704/194733324-0a65123b-e2e7-4e73-ac1a-363d872c23cf.png)

However, in my repository, renovate automerges PRs after approval review, but does not include the PR number at the end of the commit message:
![image](https://user-images.githubusercontent.com/35435704/194733402-b964836f-8391-4fde-bf8b-335640323f02.png)

I couldn't easily locate the configuration option that controls this- is it something I can control thru `renovate.json`, or would that be a GitHub setting?

Here's my `renovate.json`:
```json
{
	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
	"timezone": "America/Phoenix",
	"extends": [
		":ignoreModulesAndTests",
		":autodetectPinVersions",
		":prConcurrentLimit10",
		"group:recommended",
		"workarounds:all",
		":semanticCommitType(chore)"
	],
	"automerge": true,
	"automergeType": "pr",
	"automergeStrategy": "squash",
	"platformAutomerge": true,
	"lockFileMaintenance": { "enabled": true, "automerge": true }
}
```

**Comment by rarkins** [comment, marked as answer]
The difference is likely because we've enabled the config option platformAutomerge

**Reply by elijaholmos** [reply]
I have that option enabled too

**Reply by rarkins** [reply]
Maybe you enabled it after the PR was created. It only applies for PRs created after the option is turned on

**Reply by elijaholmos** [reply]
No, I've had this same configuration for a few weeks now

**Reply by rarkins** [reply]
GitHub chooses the title when it does platform merging

**Reply by elijaholmos** [reply]
Is there a repo setting that would create the disparity between the titles selected for your repo's platform merges vs my repo's platform merges?

**Reply by rarkins** [reply]
Maybe this? https://github.blog/changelog/2022-05-11-default-to-pr-titles-for-squash-merge-commit-messages/

**Reply by elijaholmos** [reply]
I'll give that a try and let you know, thanks

**Reply by elijaholmos** [reply]
changing the setting at the repo level solved my problem. no configuration modifications to renovate were necessary.
![image](https://user-images.githubusercontent.com/35435704/195380323-93514a0c-36b1-460d-a7d8-c7ca1060df18.png)
